kubernetes/template-generator/template_generator.py

```diff
@@ -0,0 +1,342 @@
+#!/usr/bin/env python
+
+import re
+import yaml
+import json
+
+
+class SanitizeException(Exception):
+    pass
+
+
+class TemplateGenerator(object):
+    def __init__(self, container_name, system, exposed_ports=None, external_ips=None):
+        """
+            @param string container_name    the name of migrated container
+            @param ContainerOS system       a os description module
+            @param list exposed_ports       a list of exposed ports
+        """
+        self.sanit_container_name = self.sanitize_container_name(container_name)
```

Comment:
indeed it does, since I do not see any other place that this is assigned. I think it's fine that it is sanitized but you don't need to call it that way if there's no other form of that name (which is not sanitized)